src/Monolog/Processor/PsrLogMessageProcessor.php

```diff
@@ -51,7 +51,7 @@ public function __invoke(array $record): array
             } elseif (is_object($val)) {
                 $replacements['{'.$key.'}'] = '[object '.get_class($val).']';
             } else {
-                $replacements['{'.$key.'}'] = '['.gettype($val).']';
+                $replacements['{'.$key.'}'] = 'array'.@json_encode($val);
```

Comment:
We really don't know that it is an array there? Could be a resource, or maybe other stuff I can't think of right now because it's late :p